### ISSUE TITLE
Can't run klogg.exe on Windows because of missing msvcp140_2.dll

### ISSUE THREAD

**Issue by czlitony** [variar/klogg#672]
![Snipaste_2023-08-28_13-27-43](https://github.com/variar/klogg/assets/20141496/c1c3d26d-d4c5-4e59-b1dd-8e63fd74970d)

**Comment by variar** [comment]
Is this still the case with latest builds, 24.11.0.1662+?

**Comment by daarrgcs** [comment]
No, the 24.11.0.1685 gives a series of errors related to the Qt6 libraries. 

![Image](https://github.com/user-attachments/assets/d39857fc-4996-468e-b346-474b13619604)

**Comment by daarrgcs** [comment]
Installed Qt6 Creator and DesignStudio. Rebooted but got same errors. Copied dll's to the klogg folder. (Core, Gui, Network I think) Got same errors.
